nursery/reference-doh.yml

```diff
@@ -0,0 +1,75 @@
+rule:
+  meta:
+    name: reference of DoH
```

Comment:
lets rename this to "references DNS over HTTPS endpoints"

Reply:
will need to rename the file, too

Reply:
Done!